include/tscore/Ptr.h

```diff
@@ -47,32 +46,23 @@ class RefCountObj : public ForceVFPTToTop
 {
 public:
   RefCountObj() {}
-  RefCountObj(const RefCountObj &s)
-  {
-    (void)s;
-    return;
-  }
-
+  RefCountObj(const RefCountObj &) = delete;
```

Comment:
This has to be:
```
RefCountObj(const RefCountObj &)
{
  m_refcount = 0;
}
```

Reply:
Seems like a hinky design.  The atomic ref count indicates instances of classes derived from this one should be thread-safe.  But, for that to be true, wouldn't each instance need a shared_mutex?  With read access taking a shared lock, and write access taking an exclusive lock?

Reply:
Ha! you can't escape me just by muting the hackathon audio!